src-tauri/src/behavior/farming_behavior.rs

```diff
@@ -402,7 +402,15 @@ impl<'a> FarmingBehavior<'_> {
                 send_keystroke(index.into(), KeyMode::Press);
             }
         }
-
+        // If pickup emote is set 
+        if let Some(index) = config.get_slot_index(SlotType::PickupEmote) {
+            std::thread::sleep(std::time::Duration::from_millis(self.rng.gen_range(80..100)));
```

Comment:
Is there a functional reason for waiting before starting to pick up items?

Reply:
I wait the item falls to be sure